utils/test/ArcGISProTestKickStart.bat

```diff
@@ -13,7 +13,7 @@ rem  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 rem  See the License for the specific language governing permissions and
 rem  limitations under the License.
 rem ------------------------------------------------------------------------------
-rem  TestKickStart.bat
+rem  ArcGISProTestKickStart.bat
```

Comment:
I'll create an issue here to add both tests. https://github.com/ArcGIS/military-tools-for-arcgis-build